### ISSUE TITLE
Notebook Completer UI/UX improvement

### ISSUE THREAD

**Issue by Carreau** [jupyter/jupyterlab#292]
So I like the new completer UI that highlight the matches between the current text and the completion.
There is a behavior which I'm not sure ison purpose, the highlighting of these charaters appears only after the second tab press, and the ordering of completion changes:

![matunder](https://cloud.githubusercontent.com/assets/335567/16747628/80a2e704-4785-11e6-8705-987a6e81203d.gif)

**Comment by ellisonbg** [comment]
Closing in favor of new issues about tab completer...

**Comment by Carreau** [comment]
Can you link to the new issue(s) the tracks this ?

**Comment by jasongrout** [comment]
See https://github.com/jupyter/jupyterlab/pull/673 for the implementation of the search icon.

**Comment by ellisonbg** [comment]
Sorry, new issue is in #847

**Comment by Carreau** [comment]
Thanks !
